### PR TITLE
[DOCS-8667] Add Obs Pipelines troubleshooting doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

- Adds a new Troubleshooting doc.
- Adds Troubleshooting to nav.
- Moves a section from the Set Up Pipelines doc to Troubleshooting.
- Adds link to Troubleshooting doc.
- Remove alias from legacy Troubleshooting doc.

[DOCS-8667](https://datadoghq.atlassian.net/browse/DOCS-8667)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-8667]: https://datadoghq.atlassian.net/browse/DOCS-8667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ckelner — APPROVED** [review]
Nice! Seems clear to me, thanks May!

File: content/en/observability_pipelines/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Navigate to [Observability Pipelines][2].
+1. Select your pipeline.
+1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.
+1. To view the Workers' statuses and versions, click the **Latest Deployment & Setup** tab.
+1. To see the Workers' logs, click the cog at the top right side of the page, then select **View OPW Logs**. See [Logs Search Syntax][3] for details on how to filter your logs. To see logs for a specific Worker, add `@op_work.id:<worker_id>` to the search query.
+
+## Inspect events sent through your pipeline to identify setup issues
+
+You can `tap` into the data that is going through your sources, processors, and destinations to see the raw data flowing through each individual component of your pipeline.
```
Comment (sghall) [inline content/en/observability_pipelines/troubleshooting.md:22]: 👋 @maycmlee You can't actually tap into destinations.

File: content/en/observability_pipelines/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+To view information about the Observability Pipelines Workers running for an active pipeline:
+
+1. Navigate to [Observability Pipelines][2].
+1. Select your pipeline.
+1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.
+1. To view the Workers' statuses and versions, click the **Latest Deployment & Setup** tab.
+1. To see the Workers' logs, click the cog at the top right side of the page, then select **View OPW Logs**. See [Logs Search Syntax][3] for details on how to filter your logs. To see logs for a specific Worker, add `@op_work.id:<worker_id>` to the search query.
+
+## Inspect events sent through your pipeline to identify setup issues
```
Comment (sghall) [inline content/en/observability_pipelines/troubleshooting.md:20]: I wonder if this section should start out by saying something like "If you can access your OPW instances locally..."  It jumps from stuff in the UI in `View Observability Pipelines Worker stats and logs` to this section abruptly I think.  We're now assuming the user is local to where OPW is installed.

**Review by hestonhoffman — APPROVED** [review]
LGTM

**Comment by maycmlee** [comment]
> One small note. Might be out of scope for this PR. I noticed the stuff in the legacy directory is still searchable, but there's a banner on the legacy index page that explains the versioning stuff. 

@hestonhoffman when you say the legacy directory is still searchable, do you mean you can find those docs in a Google or Algolia search? The legacy docs should all be private and not searchable: https://github.com/DataDog/documentation/blob/master/content/en/observability_pipelines/legacy/_index.md?plain=1#L18-L20

**Comment by hestonhoffman** [comment]
@maycmlee I missed that cascade value! Good stuff

**Pull request merged**